src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/boss_black_knight.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * This file is part of the TrinityCore Project. See AUTHORS file for Copyright information
+  * This file is part of the TrinityCore Project. See AUTHORS file for Copyright information
```

Comment:
Ouch. Sorry, my bad. I managed to add 1 space more than intended here. Fixup suggestion:
```suggestion
 * This file is part of the TrinityCore Project. See AUTHORS file for Copyright information
```